src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -234,6 +239,26 @@ private Object readResolve() {
         return this;
     }
 
+    /**
+     * Returns how long to delay events received from GitHub in order to allow the API caches to sync.
+     *
+     * @return how long to delay events received from GitHub in order to allow the API caches to sync.
+     */
+    public static int getEventDelaySeconds() {
+        return eventDelaySeconds;
+    }
+
+    /**
+     * Sets how long to delay events received from GitHub in order to allow the API caches to sync.
+     *
+     * @param eventDelaySeconds number of seconds to delay, will be restricted into a value within the range
+     *                          {@code [0,300]} inclusive
+     */
+    @Restricted(NoExternalUse.class) // to allow configuration from system groovy console
```

Comment:
I know it is technically unnecessary but I want to indicate intent, which the setter provides